### PR TITLE
Add Fair pointers tutorials

### PR DESCRIPTION
**Pull request by khaled196**
This pull request to add FAIR Pointers tutorials to GTN

### CODE REVIEW

File: topics/fair/tutorials/fair-metadata/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+# Rich metadata in public data repositories
+
+Help with rich metadata curation is often supported by public data repositories, and data deposition is one way you can improve its level of FAIRness.  During submission, metadata is composed and linked, making it understood, accessible and searchable.
+
+The figure below shows a screenshot of a dataset hosted by the [BioStudies](https://www.ebi.ac.uk/biostudies/) (Figure 2.2).  BioStudies is a public database holding descriptions for biological studies and their (meta)data, and is often used by researchers to provide **primary identifiers** to supplementary information described in publications.
+
+
+![BioStudies](../../images/figure2-2_rnaseq-database.png "A screenshot of a plant RNAseq dataset housed in BioStudies showing rich metadata {% cite RubénSchlaen %}")
```
Comment (hexylena) [inline topics/fair/tutorials/fair-metadata/tutorial.md:131]:
```suggestion
![A screenshot of a plant RNAseq dataset housed in BioStudies showing rich metadata like accession, study type, publication, etc.](../../images/figure2-2_rnaseq-database.png "A FAIR dataset in a public repository {% cite RubénSchlaen %}")
```

**Review by hexylena — DISMISSED** [review]
Very cool @khaled196 ! This is great to see :) Made some small comments but overall this looks fantastic

**Review by hexylena — APPROVED** [review]

**Review by bgruening — DISMISSED** [review]
Thanks a lot @khaled196

File: topics/fair/tutorials/fair-data-registration/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  funding:
+      - ELIXIR-UK-DaSH
+subtopic: pointers
+requirements:
+  - type: "internal"
+    topic_name: fair
+    tutorials:
+      - fair-intro
+      - fair-origin
+      - fair-metadata
```
[the remark this answers is not in the document]
Comment (khaled196) [reply topics/fair/tutorials/fair-data-registration/tutorial.md:57]: Thank you, Helena, 

I am working on the pathway. I am also waiting for some info from the FAIR team about the introduction.

**Comment by khaled196** [comment]
@bgruening @hexylena Thank you a lot for your time on this. I updated the pull request with the fair team additions

File: learning-pathways/fair-training.md
Comment (hexylena) [inline learning-pathways/fair-training.md:1]: this looks fantastic

**Review by hexylena — APPROVED** [review]
The formatting overall looks quite good.

**Comment by hexylena** [comment]
@khaled196 if you'd like to make any announcements through the GTN channels, [we'd welcome a 'news post' about this addition.](https://training.galaxyproject.org/training-material/faqs/gtn/gtn_news_create_post.html)

**Pull request merged**